gulpfile.js

```diff
@@ -323,7 +323,7 @@ gulp.task('watch', function() {
 
 
 // Start a server for serving frontend
-gulp.task('connect', function() {
+gulp.task('connect', ['lint'], function() {
```

Comment:
@spyshiv I have added this beacuse until `lint task `is complete, the server will not connect.
Since the lint task fails(see [here](https://github.com/Cloud-CV/EvalAI/pull/818/files#diff-b9e12334e9eafd8341a6107dd98510c9R234)) on any error & hence the `connect task` will also fail as it is dependent on `lint`.
I think this is the missing feature in the last PR. Please refer to this [link](https://github.com/Cloud-CV/EvalAI/issues/690) .  Am I correct ?